apps/hubble-stats/app/layout.tsx

```diff
@@ -21,15 +35,72 @@ export default function RootLayout({
     setIsDarkMode('light');
   }, [setIsDarkMode]);
 
+  const items: ItemProps[] = [
+    {
+      name: 'Hubble',
+      isInternal: true,
+      href: '',
+      Icon: ContrastTwoLine,
+      subItems: [
+        {
+          name: 'Bridge',
+          isInternal: false,
+          href: BRIDGE_URL,
+        },
+        {
+          name: 'Explorer',
+          isInternal: true,
+          href: '',
+        },
+        {
+          name: 'Faucet',
+          isInternal: false,
+          href: WEBB_FAUCET_URL,
+        },
+      ],
+    },
+    {
+      name: 'Tangle Network',
+      isInternal: false,
+      href: '',
+      Icon: Tangle,
+      subItems: [
+        {
+          name: 'DKG Explorer',
+          isInternal: false,
+          href: STATS_URL,
+        },
+        {
+          name: 'Homepage',
+          isInternal: false,
+          href: TANGLE_MKT_URL,
+        },
+      ],
+    },
+  ];
+
+  const footer: FooterProps = {
+    name: 'Webb Docs',
+    isInternal: false,
+    href: WEBB_DOCS_URL,
+    Icon: DocumentationIcon,
+  };
+
   return (
     <html lang="en">
       <WebbUIProvider defaultThemeMode="light">
-        <body className="bg-body dark:bg-body_dark bg-cover flex h-screen">
-          {/* Slide Navigation */}
-          <nav className="w-[280px] h-full"></nav>
-
-          {/* Main Body */}
-          <main className="flex-[1]">
+        <Head>
+          <title>Welcome to Hubble Stats!</title>
+        </Head>
```

Comment:
Use metadata here.

Reply:
@AtelyPham This was addressed in the previous [PR](https://github.com/webb-tools/webb-dapp/pull/1394/files) we merged today. Not sure why it hasn't updated here.